bin/fail2ban_digest

```diff
@@ -110,7 +110,7 @@ def db_busy_open(filename, flags, timeout):
 
 def add(db, ip):
 	db = db_busy_open(db_location + '/' + db + '.dbm', 'c', 30)
-	if db_creation_date_key not in db.keys():
+	if db_creation_date_key.encode('UTF-8') not in db.keys():
```

Comment:
What about adding "u" in front of the string at line # 37 instead?

db_creation_date_key = u'db_creation_date'

Should have the same effect should it not?

Reply:
It should. I just made the change on my system and will let it run for a bit to double check. I'll update the PR if it works as it should.